src/layer/place.js

```diff
@@ -1,10 +1,23 @@
 import * as Label from "../constants/label.js";
+import * as Color from "../constants/color.js";
+
+const labelHaloColor = [
+  "interpolate",
+  ["linear"],
+  ["zoom"],
+  4,
+  `hsla(${Color.backgroundHsl}, 0.8)`,
+  5,
+  `hsl(${Color.backgroundHsl})`,
```

Comment:
This is identical to the `backgroundFill` constant. Anyways, I think it would be fine to define two different color constants that share the same first three components. Deconstructing the CSS color syntax gets into diminishing returns.

Reply:
Good point.  I've [simplified it.](https://github.com/ZeLonewolf/openstreetmap-americana/pull/606/commits/df867e24e6aada7249a67e8e2ee128806486b709)  I guess I was thinking that the label halo color should be tied to the background color in case it changes, but that's not really necessary.